packages/app-store/locations.ts

```diff
@@ -14,6 +14,7 @@ export type DefaultEventLocationType = {
   label: string;
   messageForOrganizer: string;
   category: "in person" | "conferencing" | "other" | "phone";
+  linkType: "static";
```

Comment:
All default apps are static linkType apps. Set this explicitly so that we can access `linkType` property easily on a union of DefaultLocation and LocationFromApp